docs/intro/reusable-apps.txt

```diff
@@ -262,6 +262,22 @@ this. For a small app like polls, this process isn't too difficult.
    you add some files to it. Many Django apps also provide their documentation
    online through sites like `readthedocs.org <https://readthedocs.org>`_.
 
+   Many Python projects, including Django and Python itself, use `Sphinx
+   <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ to build
+   their documentation. If you choose to use Sphinx you can link back to the
+   Django documentation using `Intersphinx
+   <https://www.sphinx-doc.org/en/master/usage/quickstart.html#intersphinx>`_
+   by including the a value for Django in your project's
+   ``intersphinx_mapping`` configuration::
+
+        intersphinx_mapping = {
+            # ...
+            "django": (
+                "http://docs.djangoproject.com/en/stable/",
+                "http://docs.djangoproject.com/en/stable/_objects/",
```

Comment:
```suggestion
                "https://docs.djangoproject.com/en/stable/",
                "https://docs.djangoproject.com/en/stable/_objects/",
```

(I did mention that one on the call before 😁 )

Reply:
Ah, yes. 🙌